target/linux/mvebu/files/arch/arm/boot/dts/armada-370-c200-v2.dts

```diff
@@ -0,0 +1,390 @@
+		};
+
+		usb1 {
+			label = "USB1 Button";
+			linux,code = <BTN_0>;
+			gpios = <&gpio1 0 GPIO_ACTIVE_LOW>;
+		};
+
+		usb2 {
+			label = "USB2 Button";
+			linux,code = <BTN_1>;
+			gpios = <&gpio0 14 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	gpio-poweroff {
+		compatible = "gpio-poweroff";
+		pinctrl-0 = <&pmx_poweroff>;
+		pinctrl-names = "default";
+		gpios = <&gpio0 7 GPIO_ACTIVE_HIGH>;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		pinctrl-0 = <&pmx_leds1 &pmx_leds2>;
+		pinctrl-names = "default";
+
+		led_status_green: status-green {
+			label = "green:status";
+			gpios = <&gpio1 26 GPIO_ACTIVE_LOW>;
+		};
+
+		led_status_red: status-red {
+			label = "red:status";
+			gpios = <&gpio1 24 GPIO_ACTIVE_LOW>;
+		};
+
+		cloud-blue {
+			label = "blue:cloud";
+			gpios = <&gpio1 20 GPIO_ACTIVE_HIGH>;
+		};
+
+		disk1-green {
+			label = "green:disk1";
+			gpios = <&gpio1 25 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "ata1";
+		};
+
+		disk1-red {
+			label = "red:disk1";
+			gpios = <&gpio1 18 GPIO_ACTIVE_LOW>;
+		};
+
+		disk2-green {
+			label = "green:disk2";
+			gpios = <&gpio1 15 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "ata2";
+		};
+
+		disk2-red {
+			label = "red:disk2";
+			gpios = <&gpio1 19 GPIO_ACTIVE_LOW>;
+		};
+
+		disk-fail-green {
+			label = "green:disk-fail";
+			gpios = <&gpio1 23 GPIO_ACTIVE_LOW>;
+		};
+
+		disk-fail-red {
+			label = "red:disk-fail";
+			gpios = <&gpio1 21 GPIO_ACTIVE_LOW>;
+		};
+
+		usb1-green {
+			label = "green:usb1";
+			gpios = <&gpio0 16 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "usbport";
+			trigger-sources = <&usb1_port>;
+		};
+
+		usb1-red {
+			label = "red:usb1";
+			gpios = <&gpio0 15 GPIO_ACTIVE_LOW>;
+		};
+
+		usb2-green {
+			label = "green:usb2";
+			gpios = <&gpio0 13 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "usbport";
+			trigger-sources = <&usb2_port>;
+		};
+
+		usb2-red {
+			label = "red:usb2";
+			gpios = <&gpio0 12 GPIO_ACTIVE_LOW>;
+		};
+	};
+};
+
+&coherencyfab {
+	broken-idle;
+};
+
+&eth1 {
+	pinctrl-0 = <&ge1_rgmii_pins>;
+	pinctrl-names = "default";
+	status = "okay";
+	phy-handle = <&ethphy0>;
+	phy-connection-type = "rgmii-id";
+};
+
+&i2c0 {
+	pinctrl-0 = <&i2c0_pins>;
+	pinctrl-names = "default";
+	clock-frequency = <100000>;
+	status = "okay";
+
+	nct7802@2a {
+		compatible = "nuvoton,nct7802";
+		reg = <0x2a>;
+	};
+
+	rtc@30 {
+		compatible = "s35390a";
+		reg = <0x30>;
+	};
+};
+
+&mdio {
+	pinctrl-0 = <&mdio_pins>;
+	pinctrl-names = "default";
+
+	ethphy0: ethernet-phy@0 { /* Marvell 88E1318 */
+		reg = <0>;
+		#thermal-sensor-cells = <0>;
+	};
+};
+
+&nand_controller {
+	status = "okay";
+
+	nand@0 {
+		reg = <0>;
+		label = "pxa3xx_nand-0";
+		nand-rb = <0>;
+		marvell,nand-keep-config;
+		nand-on-flash-bbt;
+		nand-ecc-strength = <4>;
+		nand-ecc-step-size = <512>;
+
+		partitions {
+			compatible = "fixed-partitions";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			partition@0 {
+				label = "uboot";
```

Comment:
should some of these partitions be "read-only;"?

Reply:
done